pallets/marketplace/src/migrations/v6.rs

```diff
@@ -34,6 +34,8 @@ pub fn migrate<T: Config>() -> Weight {
                 info.owner,
                 info.allow_list,
                 "Ternoa Marketplace".into(),
+                Default::default(),
```

Comment:
Just comment out the whole file. Storage version 6 is already in use and we don't need to have a migration function from v5 to v6.